src/libraries/System.Private.CoreLib/src/System/DateTime.Windows.cs

```diff
@@ -1,30 +1,164 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Diagnostics;
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
+using System.Threading;
 
 namespace System
 {
     public readonly partial struct DateTime
     {
-        internal static readonly bool s_systemSupportsLeapSeconds = SystemSupportsLeapSeconds();
+        private static LeapSecondCache? s_leapSecondCache;
+
+        private static unsafe delegate* unmanaged[Stdcall]<ulong*, void> s_pfnGetSystemTimeAsFileTime = GetGetSystemTimeAsFileTimeFnPtr();
```

Comment:
```suggestion
        private static unsafe delegate* unmanaged<ulong*, void> s_pfnGetSystemTimeAsFileTime = GetGetSystemTimeAsFileTimeFnPtr();
```
`[Stdcall]` is not necessary. Platform default works fine. We do have explicit StdCall on Windows-specific DllImports in under libraries either.